### DISCUSSION TITLE
What is the difference between a domain id of 0 and 1

### DISCUSSION THREAD

**Discussion by zhuyu1932** [eProsima/Fast-DDS#4478] [Support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

create_participant(0, pqos) can create 100 domain participants

### Current behavior

Using the helloworld example, with domain 0, you can only create 3-5 domain participants, and with domain 1, you can break through 5. If create_participant(0, pqos) is changed to create_participant(1,pqos), the number of domain participants can exceed five.

### Steps to reproduce

helloworld

### Fast DDS version/commit

2.13.1

### Platform/Architecture

Windows 10 Visual Studio 2019

### Transport layer

Default configuration, UDPv4 & SHM

**Comment by zhuyu1932** [comment]
And start several sub end and a pub end, pub sent 10000, a sub end can receive 9000, and some sub can only receive 100.

**Comment by Mario-DL** [comment]
Hi @zhuyu1932 
Thanks for using `Fast DDS`. The report's information seems incomplete to me, what are the steps to reproduce the issue? information about the application ?, the problem is that there is no communication, or that it crashes ?...  

Besides, I think that the title is unrelated with the issue. In `DDS` using different domains is a way to isolate different parts of the system/application. Please refer to the [documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domain.html).

I am feeling free to move this thread to a discussion until it is proved that, in fact, it is an issue.

**Comment by ploe24** [comment]
@zhuyu1932 

> Using the helloworld example, with domain 0, you can only create 3-5 domain participants, and with domain 1, you can break through 5. If create_participant(0, pqos) is changed to create_participant(1,pqos), the number of domain participants can exceed five.

I could be wrong, but maybe this helps:
Depending on how and where you installed FastDDS, go to `C:\ProgramData\eprosima\fastrtps_interprocess` and clear everything in that directory.
We had a similar issue once when playing around with a local domain, where domain `0` was "blocked" somehow.
It wasn't possible to use the domain ID 0 anymore, until we deleted all mutex/port files in there.
